src/components/Disputes/DisputeDetail.js

```diff
@@ -79,43 +69,52 @@ const DisputeDetail = React.memo(function DisputeDetail({ match }) {
       <Bar>
         <BackButton onClick={handleBack} />
       </Bar>
-      {dispute?.status === DisputeStatus.Voided ? (
-        DisputeInfoComponent
-      ) : (
-        <Split
-          primary={
-            <React.Fragment>
-              {DisputeInfoComponent}
-              {(() => {
-                if (disputeFetching) {
-                  return null
-                }
-                if (evidences.length === 0) {
-                  return <NoEvidence />
-                }
-                return (
-                  // TODO- in next PR will get plaintiff and deffendant from the dispute
-                  <DisputeEvidences
-                    evidences={evidences}
-                    plaintiff={creatorAddress}
-                    defendant=""
-                  />
-                )
-              })()}
-            </React.Fragment>
-          }
-          secondary={
-            <React.Fragment>
-              <Box heading="Dispute timeline" padding={0}>
-                {disputeFetching ? (
-                  <div css="height: 200px" />
-                ) : (
-                  <DisputeTimeline dispute={dispute} />
-                )}
-              </Box>
-            </React.Fragment>
-          }
+      {noDispute ? (
+        <ErrorLoading
+          subject="dispute"
+          errors={[`Dispute #${disputeId} does not exist`]}
```

Comment:
should we check this error msg with @dizzypaty ?

Reply:
> should we check this error msg with @dizzypaty ?

@rperez89 – See designs for this error below: 

<img width="1534" alt="dispute-not-found" src="https://user-images.githubusercontent.com/7205369/75431457-c9697000-5944-11ea-96eb-0a0a25151a27.png">

Figma file: ✅ MASTER: Court Dashboard > Page: - Error pages
https://www.figma.com/proto/IdVA6iMn0cWLV5imylwEUP/%E2%9C%85-MASTER-Court-Dashboard?node-id=4158%3A52&viewport=170%2C1002%2C0.25&scaling=min-zoom